.gitignore

```
@@ -2,3 +2,4 @@
 *.swp
 node_modules
 /dist
+.idea/
```

```suggestion
/.idea
```
Can this work? Is there a meaning in trailing slash?

---

changed!